docker-compose.hybrid.yml

```diff
@@ -0,0 +1,21 @@
+version: "3"
+
+# add this below if you want this container to attach to an already created docker network
+#networks:
+  #yournetwork:
+    #external: true
+
+services:
+  hamclock:
+    image: hamclock:latest
```

Comment:
ggilman/hamclock

Reply:
actually the image that is referred to by that compose file would be the image that would be built by fallowing the instructions that are in the hybrid install section of the updated readme file.  Not sure that the hybrid compose file I wrote would work with the ggilman/hamclock image or not but give it a try and let us know

Reply:
If the hybrid compose file were written to use the image by ggilman/hamclock then it would look like this 

version: "3"
services:
  hamclock:
    image: ggilman/hamclock

(this is a sample of the compose file from the example on ggilman/hamclock page on docker hub)

Reply:
I explicitly refrained from pushing a compiled docker file anywhere since hamclock is not open source and the licensing is not made clear. I'm not compiling and distributing code that's not my own onto any platforms and violate both ethic/moral rules and potenially copyright laws. So using ggilman/hamclock would be out of the question for me.

Sure it would make this whole thing more easy to just provide a precompiled image and compile it trough a scheduled pipeline daily, but that's not what the licensing allows us to do.